SearchBar/index.js

```diff
@@ -68,11 +71,21 @@ export default class SearchBar extends React.Component {
     }
 
     onParseOk(expressions) {
+        const {
+            autoSearch,
+        } = this.props;
+
+        if(autoSearch !== false) {
+            this.processQuery(expressions);
+        }
+    }
+
+    processQuery(expressions) {
         const {
             options,
             data,
             scroll,
-            columns = false
+            columns = false,
         } = this.props;
 
         if (!isEqual(this.expressions, expressions)) {
```

Comment:
since this.processQuery is called onClick now let's move setTimeout only for the onParseOk, we want the query to be processed on user clicking the search icon.